sabnzbd/interface.py

```diff
@@ -858,6 +858,7 @@ def saveSwitches(self, **kwargs):
     "api_warnings",
     "helpful_warnings",
     "ampm",
+    "enable_multipar",
```

Comment:
The unit tests are correct. You added it here, but on the actual Config page. Also, shouldn't it be a Special setting? Then we also don't need a new skintext.

Reply:
It is a special setting, but added just like the rest where they do have a skin text entry.. are you saying they shouldn't have them?

Reply:
Oh yes sorry, it's indeed a Special. But Special settings don't have a skintext.

Reply:
yeah, plan was up in the air if we would keep multipar support or just remove. but as I dont think ill add it on config landing page like the 7zip/multipar one does.. ill drop here